templates/web/legacy/embedded_legacy_form.ftl

```diff
@@ -79,6 +79,9 @@
   var path = CStudioAuthoring.Utils.getQueryVariable(location.search, 'path');
   var site = CStudioAuthoring.Utils.getQueryVariable(location.search, 'site');
   var type = CStudioAuthoring.Utils.getQueryVariable(location.search, 'type');
+  var contentTypeId = CStudioAuthoring.Utils.getQueryVariable(location.search, 'contentTypeId');
+  var newEdit = CStudioAuthoring.Utils.getQueryVariable(location.search, 'newEdit');
```

Comment:
What's the meaning of newEdit? Is it to say it is a new piece of content instead of editing an existing?
If so, let's rename to `isNewContent`. If not, still might be worth trying to find a clearer name.